src/app/services/prices-calculation.service.ts

```diff
@@ -27,8 +27,14 @@ export class PricesCalculationService {
   private rewardEnded = new Set<string>();
   private lastPrices = new Map<string, PricesDto>();
 
-  constructor(private log: NGXLogger) {
+  constructor(private log: NGXLogger, @Inject(APP_CONFIG) public config: AppConfig, private logger: NGXLogger) {
     StaticValues.vaults.forEach(v => this.tvls.set(v, 0.0));
+    this.logger.updateConfig({ 
```

Comment:
looks like we already did it in app.ts

Reply:
I agree that might fit better in a different place. I'm not sure where exactly, but I'm happy to move it if there are any suggestions.

Unfortunately, I was unable to get the values in app.ts to work. I think it is due to the logger being an import and the APP_CONFIG value not being set at import time. I had to add the logger.updateConfig piece here to get the correct values in the logger. Prior to adding the updateConfig the logger was outputting "undefined" values due to the APP_CONFIG not containing any values at the time of import.

I can update the values in app.ts to clearly show they are placeholder values until the APP_CONFIG value is populated and can be injected in a logger.updateConfig() call

Reply:
updated the values in app.ts to indicate they are just placeholders there and that they will be updated at runtime

Reply:
hm, it should work in app.ts
I'll investigate this, thanks